src/locale/translations/fr.json

```diff
@@ -81,7 +81,11 @@
       "Body1": "Merci d’aider à ralentir la propagation de la COVID-19!",
       "Body2": "Rappel : ",
       "Body3": "Pour contribuer à la sécurité des autres, vous devez partager vos nouvelles expositions tous les jours, surtout si vous avez à sortir de la maison.",
+<<<<<<< HEAD
```

Comment:
Merge conflict to be cleaned up

Reply:
Already has been, missed that so I pushed again right after with the merge conflict fix